src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -112,38 +113,39 @@ public void actionPerformed(ActionEvent e) {
             	emailJText_.setText("");
             	
                 // Clears the Text Fields
-                Debug.log("Next action taken; For now, clears the field. Implement in Sprint 5.");
+                Debug.log("Next action taken; For now, clears the field.  Implement in Sprint 5.");
             }
         });
         
         ImageIcon addButtonIcon = ImageHelper.createImageIcon("images/add.jpg");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "addButtonIcon" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)